getgauge/util.py

```diff
@@ -12,8 +12,11 @@ def get_project_root():
 
 
 def get_step_impl_dirs():
-    STEP_IMPL_DIR_NAMES = map(str.strip, os.getenv(STEP_IMPL_DIR_ENV).split(',')) if os.getenv(STEP_IMPL_DIR_ENV) else ['step_impl']
-    return [os.path.join(get_project_root(), name) for name in STEP_IMPL_DIR_NAMES]
+    step_impl_dir_names = map(str.strip, os.getenv(STEP_IMPL_DIR_ENV).split(',')) if os.getenv(STEP_IMPL_DIR_ENV) else ['step_impl']
```

Comment:
Will great to have the test for this [here](https://github.com/getgauge/gauge-python/blob/master/tests/test_utils.py)

Reply:
Sorry I didn't have time for the tests yet, pretty busy these days. Hope to add some soon